contrib/test_jenkins.sh

```diff
@@ -574,10 +574,15 @@ run_ucx_perftest_mpi() {
 	# hack for perftest, no way to override params used in batch
 	# todo: fix in perftest
 	sed -s 's,-n [0-9]*,-n 1000,g' $ucx_inst_ptest/msg_pow2 | sort -R > $ucx_inst_ptest/msg_pow2_short
-	cat $ucx_inst_ptest/test_types | grep -v cuda | sort -R > $ucx_inst_ptest/test_types_short
+	cat $ucx_inst_ptest/test_types | grep -v cuda | grep -v ucp | sort -R > $ucx_inst_ptest/test_types_short_uct
```

Comment:
maybe split '$ucx_inst_ptest/test_types' to $ucx_inst_ptest/test_types_uct and $ucx_inst_ptest/test_types_ucp?
to not rely on test name to decide if it's ucp/uct test